roc/version.go

```diff
@@ -0,0 +1,52 @@
+package roc
+
+/*
+#include <roc/version.h>
+*/
+import "C"
+
+import (
+	"runtime/debug"
+	"sync"
+)
+
+var (
+	version     *Version
+	versionOnce sync.Once
+)
+
+// Version components.
+type Version struct {
+	Library *SemanticVersion // Native library version, librock version.
+	Binding string           // Binding version.
+}
+
+// Semantic version components.
+type SemanticVersion struct {
+	Major uint // Major version component.
+	Minor uint // Minor version component.
+	Patch uint // Patch version component.
+}
+
+// Retrieve version numbers.
+// This function can be used to retrieve actual run-time version of the library.
+// It may be different from the compile-time version when using shared library.
+func GetVersion() *Version {
```

Comment:
nit:

* we can remove the pointer to make it clear that it's never nil
* maybe rename it from GetVersion to just Version for simplicity? roc.Version() looks better IMHO
* maybe we can also rename struct Version to struct Versions to avoid confusion with the function